api/src/main/java/com/datastrato/gravitino/rel/Schema.java

```diff
@@ -5,6 +5,7 @@
 package com.datastrato.gravitino.rel;
 
 import com.datastrato.gravitino.Auditable;
+import com.datastrato.gravitino.SupportsSchemas;
```

Comment:
I think `Schema` can also be moved out of `rel` package, since messaging/fileset catalog also has the concept of `Schema`.

Reply:
OK, will move it out of rel package, together with SchemaChange.java

Reply:
done, moved out of rel package.